Scheduler/observing_blocks_auxtel/BLOCK-273.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "BLOCK-273",
+  "program": "BLOCK-273",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "auxtel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "target_name": "PN Hen 2-113"
+      }
+    },
+    {
+        "name": "auxtel/correct_pointing.py",
+        "standard": false,
+        "parameters": {
+            "el": "$alt",
+            "az": "$az",
+            "mag_limit": 6.0,
+            "mag_range": 4.0,
+            "radius": 10.0
+        }
+      },
+      {
+        "name": "auxtel/track_target.py",
+        "standard": true,
+        "parameters": {
+          "target_name": "PN Hen 2-113"
```

Comment:
Instead of tracking the target by name, you should receive the target coordinates from the scheduler. See how we do this in the [spectroscopic survey](https://github.com/lsst-ts/ts_config_ocs/blob/develop/Scheduler/observing_blocks_auxtel/survey-spec.json).